cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/server.go

```diff
@@ -40,6 +37,24 @@ const (
 	fluxHelmRepository     = "helmrepository"
 	fluxHelmRepositories   = "helmrepositories"
 	fluxHelmRepositoryList = "HelmRepositoryList"
+	fluxHelmChart          = "HelmChart"
+	fluxHelmCharts         = "helmcharts"
+)
+
+// these should really be constants, rather than global vars
+// but alas, go does not allow const structs (why???)
```

Comment:
Because an instance of a struct is not a compile-time object, therefore, not really constant. I assume the reason for that is for the speed of the compiler (it does not need to check whether code is updating things that are declared as constants).

I'd personally prefer not to set a precedent of using global vars at all, and instead just instantiate the GroupVersionResource using the const strings that you already have.

Reply:
ok, sounds good. I'll change this. 
FWIW, there was a lot of debate on the subject of const structs in go. What I got from it, it is technically possible just requires a lot of work, and there wasn't enough "bang for the buck" (literally an expression someone at google used) to make it happen. bleh